autoload/airline/section.vim

```diff
@@ -13,6 +12,7 @@ function! s:wrap_accent(part, value)
 endfunction
 
 function! s:create(parts, append)
+  let s:spc = g:airline_symbols.pad_sep
```

Comment:
this shouldn't be prefixed with `s:` since it's no longer a script variable, just `let spc` is sufficient to mark it as a local variable of the function.